lib/web/scripts/node-join/install.sh

```diff
@@ -40,6 +40,8 @@ TARGET_HOSTNAME="{{.hostname}}"
 TARGET_PORT="{{.port}}"
 JOIN_TOKEN="{{.token}}"
 JOIN_METHOD="{{.joinMethod}}"
+JOIN_METHOD_COMMAND=""
```

Comment:
Nit: I think `JOIN_METHOD_FLAG` and `LABELS_FLAG` would be a bit more suitable names for these variables.

Reply:
Good call, updated.